Lib/multiprocessing/heap.py

```diff
@@ -211,7 +211,10 @@ def free(self, block):
         # synchronously sometimes later from malloc() or free(), by calling
         # _free_pending_blocks() (appending and retrieving from a list is not
         # strictly thread-safe but under cPython it's atomic thanks to the GIL).
-        assert os.getpid() == self._lastpid
+        if os.getpid() != self._lastpid:
+            raise ValueError(
```

Comment:
Neither this one.

Reply:
Ditto to the above.

Reply:
Ow. I should have been more explicit because now I don't remember what I meant here. Sorry :-(

Reply:
As an absent-minded professor, I understand fully :-}